src/util/sortByLastUpdated.js

```diff
@@ -0,0 +1,8 @@
+/** returns array of recently edited data sorted by lastUpdated field */
+export const sortByLastUpdated = (recentlyEditedArray) => {
```

Comment:
Timestamps in the format `2019-12-25T20:37:39.091Z` are lexicographically comparable, thus they can be compared directly rather than converting to dates.

Reply:
e.g. `recentlyEditedArray.sort(makeCompareByProp('lastUpdated'))`